extensions/standard-processors/processors/ExecuteProcess.h

```diff
@@ -51,7 +51,7 @@ namespace processors {
 #ifndef WIN32
 
 // ExecuteProcess Class
-class ExecuteProcess : public core::Processor {
+class ExecuteProcess : public core::Processor, public core::annotation::input::Forbidden {
```

Comment:
I think this should allow - the process to be executed can be speficied in flowfile attributes and whatnot

Reply:
it can be specified but it seems to me, that it won't be used, this might be a bug

Reply:
Input is also forbidden in NiFi, I will ask them what they think about this.

Reply:
ExecuteStreamCommand is the one in NiFi that accepts commands from an incoming flowfile. We don't have that (yet?).

Reply:
Yep, makes sense to differentiate the two, but that's definitely out of scope of this change. 
Leave it as "allowed" for now, later we can align the processors to NiFi

Reply:
Done.